### PR TITLE
add random number generator and log improvement

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
- add a random number generator. We could use this with a boundary value.
- improve the watcher log
- add configuration of linter

### CODE REVIEW

File: random.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package gocommon
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+func GenRandNumber(boundary int64) (int64, error) {
```
Comment (bk201) [inline random.go:8]: Please help add a comment for the doc. It should describe what it does and the boundary (inclusive or exclusive).

Comment (Vicente-Cheng) [reply random.go:11]: Hi @bk201, @ibrokethecloud
Update the comment for the document. Please help to recheck it!
Thanks

File: random.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package gocommon
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+// GenRandNumber can generate a random number with a positive boundary value.
```
Comment (bk201) [inline random.go:8]:
```suggestion
// GenRandNumber generates a random positive number less than the boundary value.
```
I guess we can remove "can", the rest lgtm.

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Review by ibrokethecloud — APPROVED** [review]
LGTM. thanks.

**Pull request merged**